docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -678,9 +689,120 @@ This could be done by assigning unique (or practically unique, with very low pro
+found in ACS PUMS recent immigrants.
+
+Then, we distribute emigration within each race/ethnicity and move type by further demographics,
+according to the distributions of these demographics in a resample of the corresponding ACS immigrant population,
+with perturbation as described in the :ref:`perturbation section below <census_prl_perturbation>`.
+Note that in the case of household moves, these are the demographics **of the immigrant's household's reference person**,
+while for the other two types they are demographics of the immigrant themselves.
+
+Finally, we calculate the rates of people emigrating per year of person-time "at risk":
+
+* The "at risk" population for household moves is people living in non-GQ households.
+* The "at risk" population for GQ person moves is people living in GQ.
+* The "at risk" population for non-reference-person moves is people living in non-GQ households who
+  are not the reference person in their household.
+
+In order to mitigate the sampling noise in ACS PUMS stratified by all of these demographic characteristics,
+we calculate the denominator for the rate from a resample of the "at risk" population, with perturbation.
+
+Simulation strategy
+^^^^^^^^^^^^^^^^^^^
+
+Emigration events are modeled as happening to an at-risk population at a certain rate.
+They are constant across time in the simulation.
+
+Households and individuals selected to have emigration events should remain in the simulation, but their
+state and PUMA attributes should be set to a placeholder value that signifies they are
+no longer in the US.
+Emigrating simulants should also have their address attribute removed, and they should terminate
+employment.
+In the future, we may want to sample some of these simulants for re-entry in the immigration component, but for now
+they will remain outside the US permanently.
+
+Household moves
+'''''''''''''''
+
+The at-risk population for household moves is all simulants living in non-GQ households in the US.
+This at-risk population should be stratified by age group, sex, race/ethnicity, and nativity (born in or outside the US)
+**of the simulant's household's reference person**, as well as US state.
+On each time step, within each stratum, the corresponding household move emigration rate **per year of person-time** should be applied to determine
+a number of **simulants** to emigrate as part of household moves.
+Then, households within the stratum should be sampled at random for emigration until the desired number of simulants is reached.
+
+GQ person moves
+'''''''''''''''
+
+The at-risk population for GQ person moves is all simulants living in GQ in the US.
+This at-risk population should be stratified by age group, sex, race/ethnicity, nativity (born in or outside the US),
+and US state.
+On each time step, within each stratum, the corresponding GQ person move emigration rate **per year of person-time**
+should be applied to sample simulants to emigrate. 
```

Comment:
Are you proposing to have the same rate for GQ people, regardless of GQ type?

Reply:
Yes, after stratifying by demographics. Using the data, we could split this by institutional/non-institutional.

Should I split it, or add this as a limitation? It seems likely to me that the GQ type (institutional vs non) has a big effect on emigration rates, but it's not clear that this would have substantial PRL implications.